src/searchbar.cpp

```diff
@@ -208,16 +208,16 @@ void SearchBarLineEdit::onScroll(int value)
         return;
     }
 
-    /* m_completer.popup()->currentIndex() being invalid means the user used the
+    /* m_suggestionView->currentIndex() being invalid means the user used the
        keyboard up/down to scroll past the end.
     */
-    if (!m_completer.popup()->currentIndex().isValid())
-        return onScrollPastEnd();
+    if (!m_suggestionView->currentIndex().isValid())
+        onScrollPastEnd();
 
-    const auto suggestionScroller = m_completer.popup()->verticalScrollBar();
-    const auto scrollMin = suggestionScroller->minimum();
-    const auto scrollMax = suggestionScroller->maximum();
-    const bool scrolledToEnd = value == suggestionScroller->maximum();
+    auto suggestionScroller = m_suggestionView->verticalScrollBar();
+    auto scrollMin = suggestionScroller->minimum();
+    auto scrollMax = suggestionScroller->maximum();
+    bool scrolledToEnd = value == suggestionScroller->maximum();
```

Comment:
The value brought by `const`s should be long lasting rather than ephemeral (limited to a few commits).

Reply:
Done